app/models/policy.rb

```diff
@@ -56,6 +60,18 @@ def summary
     schema&.dig('summary')
   end
 
+  def readonly_attributes_changed?
+    (changed_attributes.keys & READONLY_ATTRIBUTES.map(&:to_s)).any?
+  end
+
+  def in_use?
+    account.proxies.any? { |proxy| proxy.find_policy_config_by name: name_was, version: version_was }
+  end
+
+  def idle?
```

Comment:
We don't really need this method.

Reply:
I agree 😄

Reply:
I won't touch the PR only because of this. If we need to do some more significant refactoring elsewhere, then I remove this method.

Reply:
makes sense 😉